client/stylesheets/_responsive.scss

```diff
@@ -198,6 +277,12 @@
 
 }
 
+@media (max-width: 690px) {
+    .searchResultsCardImg {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardImg` should be written in lowercase with hyphens